nextflow_schema.json

```
@@ -392,6 +393,7 @@
                     "description": "Maximum amount of memory that can be requested for any single job.",
                     "default": "128.GB",
                     "fa_icon": "fas fa-memory",
+                    "pattern": "\\d+\\.?\\s*[KMGT]?B?",
```

As mentioned in https://github.com/nf-core/tools/issues/793, maybe we should force people to use at least GB here...

```suggestion
                    "pattern": "\\d+\\.?\\s*[GT]B",
```